src/index.html

```diff
@@ -7,8 +7,33 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap">
+    <link rel="stylesheet" href="images/logo.png">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="header__logo">
+        <a href="#home">
+          <img src="images/logo.png" alt="Logo">
+        </a>
+      </div>
+      <nav>
+        <ul class="nav__list">
+          <li><a href="#apple" class="nav__link is-active apple">APPLE</a></li>
```

Comment:
```suggestion
          <li><a href="#apple" class="nav__link is-active apple">apple</a></li>
```
![Знімок екрана 2023-08-23 о 11 07 29](https://github.com/mate-academy/layout_moyo-header/assets/106310796/c5b6715c-b830-4e56-b0bf-96278c60297f)

Reply:
Not fixed